client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -25,35 +25,24 @@ const ManageAccountView = () => {
     data: { licenses, integrations }
   } = useSelector(state => state.profile);
   const dispatch = useDispatch();
-  const welcomeMessages = useSelector(state => state.welcomeMessages);
-  const onDismissWelcome = section => {
-    const newMessagesState = {
-      ...welcomeMessages,
-      [section]: false
-    };
-    dispatch({ type: 'SAVE_WELCOME', payload: newMessagesState });
-  };
   useEffect(() => {
     dispatch({ type: 'GET_PROFILE_DATA' });
   }, [dispatch, isLoading]);
   return (
-    <Container fluid className="manage-account-wrapper">
-      <Container fluid className="manage-account-content">
-        <Alert
-          isOpen={welcomeMessages.profile}
-          toggle={() => onDismissWelcome('profile')}
-          color="secondary"
-          className="welcomeMessageGeneral"
-        >
-          This page allows you to manage your account profile, change your
-          password and view software licenses.
-        </Alert>
-        <Row className="manage-account-header">
-          <h5>Manage Account</h5>
-          <Link to="/workbench/dashboard" style={{ fontWeight: '500' }}>
-            Back to Dashboard
-          </Link>
-        </Row>
+    /* !!!: Temporary bad indentation to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
+    <Section
```

Comment:
__Layout Defect__
This page was converted, but not well. It has issues that should be resolved.

<details><summary>Why Was this Not Converted Well?</summary>

Its layout was _too complex_, and I did not yet have the skill to do it (CSS Grid). Dashboard was the closest applicable layout, but the layout code for Dashboard (CSS Grid) is barely maintainable (again, past skill limitation), and should not be re-used. I have the skill to do it now, but maybe not the time.

</details>

Reply:
Done: 6ef5410